src/DynamoMigratorExtension.cs

```diff
@@ -35,20 +36,45 @@ public string Name
             }
         }
 
-        private string selectedDirectory = "No directory selected";
+        private string selectedSourceDirectory = "No directory selected";
         /// <summary>
-        /// The selected directory of dynamo files to be migrated.
+        /// The selected source directory of dynamo files to be migrated.
         /// </summary>
-        public string SelectedDirectory
+        public string SelectedSourceDirectory
         {
             get
             {
-                return this.selectedDirectory;
+                return this.selectedSourceDirectory;
             }
             set
             {
-                selectedDirectory = value;
-                RaisePropertyChanged(nameof(SelectedDirectory));
+                selectedSourceDirectory = value;
+                RaisePropertyChanged(nameof(SelectedSourceDirectory));
+            }
+        }
+
+        public string selectedTargetDirectoryDefault
+        {
+            get
+            {
+                return "Optional";
```

Comment:
what about something like:
"No target directory selected, files will be overwritten in source directory" ?